unit-tests/run-unit-tests.py

```diff
@@ -567,6 +586,17 @@ def test_wrapper( test, configuration = None ):
     finally:
         log.debug_unindent()
 
+if list_tags:

```

Comment:
I think this should be after the check for no unit-tests found

Reply:
Done